pbpy/pbtools.py

```diff
@@ -198,36 +197,32 @@ def resolve_conflicts_and_pull():
     # Make sure upstream is tracked correctly
     pbgit.set_tracking_information(pbgit.get_current_branch_name())
 
-    pblog.info("Trying to stash the local work...")
-    output = subprocess.getoutput(["git", "stash"])
-    pblog.info(str(output))
-
     pblog.info(
         "Trying to rebase workspace with latest changes on the repository...")
-    output = subprocess.getoutput(["git", "pull", "--rebase"])
+    output = subprocess.getoutput("git pull --rebase --autostash")
```

Comment:
This will change output log of git in some situations, did you test each scenario above seperated with if conditions? Without extensively testing that, I can't approve that change

Reply:
"WIP because autostash and prune features are untested."